gnat2goto/driver/tree_walk.adb

```diff
@@ -1419,6 +1417,28 @@ package body Tree_Walk is
       end case;
    end Do_Expression;
 
+   function Do_In (N : Node_Id) return Irep is
+      Left_Op : constant Irep := Do_Expression (Left_Opnd (N));
```

Comment:
Does GNAT guarantee that this node will have lower and upper bounds like this?  For example with subtype inclusion?

Reply:
I'm not entirely sure. I've added a check (to `Do_Expression`) that reports as unsupported feature if we ended up calling `Do_In` with a node that doesn't have the bounds.